lib/OpenQA/Worker/Jobs.pm

```diff
@@ -530,7 +530,12 @@ sub upload_status(;$) {
         $status->{log}             = log_snippet("$pooldir/autoinst-log.txt",   \$log_offset);
         $status->{serial_log}      = log_snippet("$pooldir/serial0",            \$serial_offset);
         $status->{serial_terminal} = log_snippet("$pooldir/virtio_console.log", \$serial_terminal_offset);
-        print STDERR $status->{serial_terminal}->{data};
+        if ($status->{serial_terminal}->{data}) {
+            print STDERR $status->{serial_terminal}->{data};
+        }
+        else {
+            print STDERR "Unable to find $pooldir/virtio_console.log\n";
+        }
```

Comment:
I'm not sure whether it is an error when the file is not present. If not, then it would make sense to ignore it.